Prace_domowe/Praca_domowa2/Karol_Degorski/pd2.ipynb

```diff
@@ -0,0 +1,3596 @@
+{
```

Comment:
<p>Nie do końca o to chodziło. Chodziło o porównanie danych imputowanych z oryginalnymi, a Ty porównałeś wszystkie dane (w tym oryginalne) z oryginalnymi. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>